hello-ethernet/defconfig

```diff
@@ -0,0 +1,33 @@
+CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_NET_L2_ETHERNET=y
+# CONFIG_NET_IPV6 is not set
+CONFIG_PINCTRL=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_ENTROPY_STM32_RNG=y
+CONFIG_LOG_BACKEND_GOLIOTH=y
+CONFIG_GOLIOTH=y
+CONFIG_GOLIOTH_SYSTEM_CLIENT=y
+CONFIG_GOLIOTH_SYSTEM_CLIENT_PSK_ID="stm32f405-feather-id@ttgo-demo"
+CONFIG_GOLIOTH_SYSTEM_CLIENT_PSK="cfa0c9537863f6268d86088376e62e0b"
```

Comment:
Some credentials are leaking here.

Reply:
Thanks for catching, this file shouldn't be in here at all so I removed it.